tests/rptest/services/redpanda.py

```diff
@@ -493,6 +494,11 @@ def __init__(self,
         self._tls_cert = None
         self._init_tls()
 
+    def set_installer(self, installer):
```

Comment:
Instead of a setter what about passing a boolean to RedpandaService that creates the installer automatically?

Reply:
The call to `RedpandaInstaller.start()` is what actually changes the initial paths on the nodes to be /opt/redpanda. Before that, we use the old style of paths (see `find_binary()`). I don't think it'd make the API much cleaner since we'd still need some `installer_started()` method here, though I can make that change if you feel strongly about it.

These should be private anyway to indicate to day-to-day test writers that they probably shouldn't be using this.